README.md

```diff
@@ -82,7 +86,6 @@ jobs:
           test: ""
           exclude_test: ""          
           extra_args: ""
-          need_secret: ""
```

Comment:
The type of the options like jobs, use_cmakelists and so on should fit the type we expect : jobs are number, use_cmakelists boolean, and for example test and exclude_test are lists.